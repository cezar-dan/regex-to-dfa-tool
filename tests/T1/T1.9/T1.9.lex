 
SPACE
0
0,' ',1
1,' ',2
1

abcde
TOKEN1
0
0,'a',1
0,'b',2
0,'c',3
0,'d',4
0,'e',5
1,'a',5
1,'b',5
1,'c',3
1,'d',4
1,'e',5
2,'a',5
2,'b',2
2,'c',3
2,'d',4
2,'e',5
3,'a',5
3,'b',5
3,'c',3
3,'d',5
3,'e',5
4,'a',5
4,'b',5
4,'c',5
4,'d',5
4,'e',6
6,'a',5
6,'b',5
6,'c',5
6,'d',5
6,'e',5
2 6 3 1 0

abcde
TOKEN2
0
0,'a',1
0,'b',2
0,'c',3
0,'d',2
0,'e',2
1,'a',6
1,'b',4
1,'c',2
1,'d',2
1,'e',2
3,'a',2
3,'b',2
3,'c',2
3,'d',2
3,'e',2
4,'a',5
4,'b',2
4,'c',3
4,'d',2
4,'e',2
5,'a',2
5,'b',4
5,'c',2
5,'d',2
5,'e',2
6,'a',2
6,'b',2
6,'c',2
6,'d',7
6,'e',2
7,'a',2
7,'b',2
7,'c',2
7,'d',2
7,'e',8
8,'a',2
8,'b',2
8,'c',2
8,'d',2
8,'e',2
3 8

cdef
TOKEN3
0
0,'c',1
0,'d',2
0,'e',3
0,'f',3
1,'c',1
1,'d',2
1,'e',3
1,'f',3
2,'c',3
2,'d',3
2,'e',4
2,'f',3
4,'c',1
4,'d',2
4,'e',3
4,'f',5
5,'c',1
5,'d',2
5,'e',3
5,'f',5
5 4 1 0

abce
TOKEN4
0
0,'a',1
0,'b',2
0,'c',3
0,'e',4
1,'a',1
1,'b',2
1,'c',3
1,'e',3
2,'a',1
2,'b',2
2,'c',3
2,'e',3
4,'a',3
4,'b',3
4,'c',5
4,'e',3
5,'a',1
5,'b',2
5,'c',3
5,'e',4
2 1

abcde
TOKEN5
0
0,'a',1
0,'b',2
0,'c',3
0,'d',4
0,'e',5
1,'a',1
1,'b',2
1,'c',3
1,'d',4
1,'e',5
2,'a',1
2,'b',2
2,'c',3
2,'d',4
2,'e',5
3,'a',1
3,'b',2
3,'c',3
3,'d',4
3,'e',5
4,'a',1
4,'b',2
4,'c',3
4,'d',4
4,'e',5
5,'a',1
5,'b',2
5,'c',3
5,'d',4
5,'e',5
5 4 3 2 1 0