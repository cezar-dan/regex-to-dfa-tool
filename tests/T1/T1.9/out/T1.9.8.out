TOKEN5 eccbbaa
SPACE  
TOKEN5 aabbecc
SPACE  
TOKEN2 aade
SPACE  
TOKEN1 bbcc
SPACE  